src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1853,11 +1853,11 @@ def __init__(self,
                  config: Optional[Dict[str, str]] = None,
                  networks: Optional[List[str]] = None,
                  placement: Optional[PlacementSpec] = None,
-                 disable_https: Optional[bool] = False,
+                 ssl: Optional[bool] = True,
                  enable_auth: Optional[bool] = False,
                  port: Optional[int] = None,
-                 ssl_certificate: Optional[str] = None,
```

Comment:
I think you need to update the mgmt-gateway teuthology test to use these new parameters
```
Error EINVAL: ServiceSpec: __init__() got an unexpected keyword argument 'ssl_certificate'
```
from https://pulpito.ceph.com/adking-2025-04-02_03:02:49-orch:cephadm-wip-adk-testing-2025-04-01-2010-distro-default-smithi/8220823/

Reply:
@adk3798 good catch! I fixed the test :+1: